CONTRIBUTING.md

```diff
@@ -35,11 +49,21 @@ Then clone the repository and install NPM packages:
 
 ### Run
 
+You made a change to the code, now you want to test it.
+
+#### Running in `Debug` mode
```

Comment:
I really try to avoid 4 levels of headings. It means there is too much nesting, it's a sign that more grouping is needed, or less granularity.